src/main/java/com/school/project/controller/UserController.java

```diff
@@ -21,9 +23,20 @@
     @Autowired
     UserService userService;
 
+    @Autowired
+    UserAccountService userAccountService;
+
     @PostMapping("/users")
-    public void createUser(@RequestBody UserDto userDto) {
-        userService.create(convertUserDtoToUser(userDto));
+    public ResponseEntity<UserDto> createUser(@RequestBody UserDto userDto) {
+        return ResponseEntity.ok()
+                .body(convertUserToUserDto(userService
+                        .create(convertUserDtoToUser(userDto))));
+    }
+
+    @PutMapping("/users/userAccount/{id}")
+    public ResponseEntity updateUserAccountType(@PathVariable long id, @RequestBody UserAccountType userAccountType){
```

Comment:
we should use parameter here, there won't be a UserAccountType on FE. You have two options, use id or parse string representation

Reply:
you mean i must use here in parameters just ID?

Reply:
id for account type, yes, correct